clippy_lints/src/lib.rs

```diff
@@ -1107,7 +1107,7 @@ pub fn register_plugins(store: &mut rustc_lint::LintStore, sess: &Session, conf:
     store.register_late_pass(|| box inherent_impl::MultipleInherentImpl::default());
     store.register_late_pass(|| box neg_cmp_op_on_partial_ord::NoNegCompOpForPartialOrd);
     store.register_late_pass(|| box unwrap::Unwrap);
-    store.register_late_pass(|| box duration_subsec::DurationSubsec);
+    store.register_late_pass(|| box manual_duration_calcs::ManualDurationCalcs);
```

Comment:
You also have to add a `register_renamed` here.

Reply:
Ok. I'll fix it